images/build/cross/Makefile

```diff
@@ -27,8 +27,8 @@ IMGNAME = kube-cross
 # Example:
 # - v1.100.0-go1.17-bullseye.0 satisfies SemVer regex, while:
 # - v1.100-go1.17-bullseye.0 does not
-KUBERNETES_VERSION ?= v1.22.0
-GO_VERSION ?= 1.17.4
+KUBERNETES_VERSION ?= v1.23.0
```

Comment:
updated here as well to the newest release, not sure if that is ok, just to confirm

Reply:
https://github.com/kubernetes/release/pull/2351#pullrequestreview-828795810